AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponentsLevel_DisplayMapperAdded.py

```diff
@@ -98,20 +157,202 @@ def AtomEditorComponentsLevel_DisplayMapper_AddedToEntity():
             display_mapper_component.get_component_property_value(
                 AtomComponentProperties.display_mapper('LDR color Grading LUT')) == display_mapper_asset.id)
 
-        # 5. Set Enable LDR color grading LUT property True
+        for operation_type in DISPLAY_MAPPER_OPERATION_TYPE.keys():
+            # 5. Set the Display Mapper Operation Type enumerated in DISPLAY_MAPPER_OPERATION_TYPE for each type
+            # Type property cannot be set currently. as a workaround we are calling an ebus to set the operation type
+            # display_mapper_component.set_component_property_value(
+            #     AtomComponentProperties.display_mapper('Type'), DISPLAY_MAPPER_OPERATION_TYPE[operation_type])
+            render.DisplayMapperComponentRequestBus(
+                bus.Broadcast, "SetDisplayMapperOperationType", DISPLAY_MAPPER_OPERATION_TYPE[operation_type])
+            general.idle_wait_frames(3)
+            set_type = render.DisplayMapperComponentRequestBus(bus.Broadcast, "GetDisplayMapperOperationType")
+            Report.info(f"DiplayMapperOperationType: {set_type}")
+
+            # 6. Set Enable LDR color grading LUT property True
+            display_mapper_component.set_component_property_value(
+                AtomComponentProperties.display_mapper('Enable LDR color grading LUT'), True)
+            Report.result(
+                Tests.enable_ldr_color_grading_lut,
+                display_mapper_component.get_component_property_value(
+                    AtomComponentProperties.display_mapper('Enable LDR color grading LUT')) is True)
+
+
+            # 7. Set Override Defaults to False
+            display_mapper_component.set_component_property_value(
+                AtomComponentProperties.display_mapper('Override Defaults'), False)
+            Report.result(
+                Tests.override_defaults,
+                display_mapper_component.get_component_property_value(
+                    AtomComponentProperties.display_mapper('Override Defaults')) is False)
+            # Set Override Defaults to True
```

Comment:
nit: can we add blank line between this comment and the code above it?

Reply:
updated